documentation/getting-started-intellij-track/building-a-scala-project-with-intellij-and-sbt.markdown

```diff
@@ -0,0 +1,94 @@
+# Building a Scala project with IntelliJ and sbt
+
+In this tutorial, we'll see how to build a Scala project [sbt](http://www.scala-sbt.org/0.13/docs/index.html). sbt is the
+de facto tool for compiling, running, and testing Scala projects of any
```

Comment:
I'm a bit concerned here about giving the impression that sbt is your only choice and e.g. maven and gradle aren't also options. Suggested:

* change "de facto" to "a common, standard"
* change "It becomes essential" to "Using a build tool becomes essential"
* add some words like "Maven, Gradle, and other popular build tools can also build Scala projects."